### PR TITLE
fix(RHINENG-12212): add CONSOLE_HOSTNAME to the correct deploy sections

### PR DESCRIPTION
**Pull request by FabriciaDinizRH**
# Overview

This PR is being created to address [RHINENG-12212](https://issues.redhat.com/browse/RHINENG-12212).

- add CONSOLE_HOSTNAME to mq sections

## PR Checklist

- [x] Keep PR title short, ideally under 72 characters
- [ ] Descriptive comments provided in complex code blocks
- [ ] Include raw query examples in the PR description, if adding/modifying SQL query
- [ ] Tests: validate optimal/expected output
- [ ] Tests: validate exceptions and failure scenarios
- [ ] Tests: edge cases
- [ ] Recovers or fails gracefully during potential resource outages (e.g. DB, Kafka)
- [ ] Uses [type hinting](https://docs.python.org/3/library/typing.html), if convenient
- [ ] Documentation, if this PR changes the way other services interact with host inventory
- [ ] Links to related PRs

## Secure Coding Practices Documentation Reference

You can find documentation on this checklist [here](https://github.com/RedHatInsights/secure-coding-checklist).

## Secure Coding Checklist

- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [ ] General Coding Practices

### CODE REVIEW

File: deploy/clowdapp.yml [hunk 1]
```diff
@@ -1388,7 +1394,7 @@ parameters:
 - name: PATH_PREFIX
   value: api
 - name: CONSOLEDOT_HOSTNAME
-  value: 'localhost'
+  value: console.stage.redhat.com
```
[the remark this answers is not in the document]
Comment (jpramos123) [reply deploy/clowdapp.yml:1397]: I will keep it as `localhost` and change the variable name in stage deployment file...

@kruai thoughts?

[the remark this answers is not in the document]
Comment (kruai) [reply deploy/clowdapp.yml:1397]: I would keep it as `localhost` as well; then if we see `localhost` being used in one of those payloads, we'll know that the env config is wrong

Comment (jpramos123) [inline deploy/clowdapp.yml:1397]:
```suggestion
  value: localhost
```

**Review by jpramos123 — CHANGES_REQUESTED** [review]
Just one suggestion

Thanks

**Review by thearifismail — APPROVED** [review]

**Review by kruai — APPROVED** [review]
LGTM!

**Review by jpramos123 — APPROVED** [review]

**Pull request merged**